content/de/ecosystem/contribution/contribution.md

```diff
@@ -57,9 +57,45 @@ lazy-src="images/ecosystem/contribution/preview_github_file_link.png" ></v-img>
 
 ### Spezielle Komponenten
 
+#### DwfIntegrationArtifact
+
+Über die Komponente `DwfIntegrationArtifact` können auf der Übersichts Seite für die Integrationsartefakte "Kacheln" 
+erstellt werden. 
+
+``` html
+<dwf-integration-artifact 
+   icon="mdi-email-fast"
+   name="mail-Integration" 
+   description="Artifact to integrate an e-mail server, so you can send mails from processes."
+   github="https://github.com/it-at-m/digiwf-email-integration"
+   docs="/resources/modules/emailintegration">
+</dwf-integration-artifact>
+```
+
+Attribute:
+- `icon` = Der [Material Design Icons](https://materialdesignicons.com/) Code für das Icon, das oben links in der Ecke dargestellt werden soll.
+- `name` = Der Name des Integrationsartefakts.
+- `description` = Die Beschreibung des Integrationsartefakts. Bitte darauf achten, dass die Beschreibung möglichst 
```

Comment:
Wäre es hier nicht schöner, wenn die description innerhalb von dwf-integration-artifact nach Zeichen begrenzt wird? Eine computed variable die einfach nur die ersten x Zeichen zurückgibt, ggf. basierend auf der Monitorgröße. Dass die Beschreibung Inhalte aus der Kachel pressen kann sollte nicht passieren.

Reply:
Mit der Monitor Größe hat das nichts zu tun. Ich hab mir das tatsächlich auch gedacht  - hab es aber als Over-Engineering verworfen. Wir haben das ja selbst in der Hand.

Reply:
Also ich finde nicht, dass eine Komponente overengineered ist wenn man die anzeigbaren Zeichen auf den tatsächlich verfügbaren Platz beschränkt ;) Finde es eher underengineered dass man so ein "technisches" Detail in die Doku schreiben muss. Aber wir könnens gern erstmal so lassen, wir schauen einfach mal wie wir damit fahren, sowas ist ja super schnell eingebaut.